src/Compilers/Core/Portable/MetadataReader/PEModule.cs

```diff
@@ -1180,6 +1180,23 @@ internal bool HasRefSafetyRulesAttribute(EntityHandle token, out int version, ou
             return false;
         }
 
+        internal bool HasMemorySafetyRulesAttribute(EntityHandle token, out int version, out bool foundAttributeType)
+        {
+            AttributeInfo info = FindTargetAttribute(MetadataReader, token, AttributeDescription.MemorySafetyRulesAttribute, out foundAttributeType);
```

Comment:
Do we have a test where a custom attribute allowing multiple was applied? I think we only consider the first usage